go/vt/tabletserver/planbuilder/dml.go

```diff
@@ -29,6 +29,11 @@ func analyzeUpdate(upd *sqlparser.Update, getTable TableGetter) (plan *ExecPlan,
 		return nil, err
 	}
 
+	// Store the WHERE clause as string for the transaction serializer.
+	buf := sqlparser.NewTrackedBuffer(nil)
+	buf.Myprintf("%v", upd.Where)
```

Comment:
See the code in tabletserver.go. I'm getting the table name separately. This way I can check for an empty table in the query.